opal/mca/threads/wait_sync.h

```diff
@@ -76,12 +76,12 @@ typedef struct ompi_wait_sync_t {
         opal_thread_internal_mutex_destroy(&(sync)->lock);     \
     }
 
-#define WAIT_SYNC_SIGNAL(sync)                              \
-    if (opal_using_threads()) {                             \
-        opal_thread_internal_mutex_lock(&(sync->lock));     \
-        opal_thread_internal_cond_signal(&sync->condition); \
-        opal_thread_internal_mutex_unlock(&(sync->lock));   \
-        sync->signaling = false;                            \
+#define WAIT_SYNC_SIGNAL(sync)                                \
+    if (opal_using_threads()) {                               \
+        opal_thread_internal_mutex_lock(&(sync)->lock);       \
+        opal_thread_internal_cond_signal(&(sync)->condition); \
```

Comment:
The new parenthesis are more confusing than helpful here.

Reply:
Thanks.  You're right.  I added one more parenthesis to unify and correct the format (see the last commit).  This clarifies which clause "&" is applied to.